src/pages/GitHubPack/index.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from "react";
+import { useSelector } from "react-redux";
+import axios from "axios";
+import { METHODS } from "../../services/api";
+import { toast } from "react-toastify";
+
+toast.configure();
+
+function GitHubStudyPack(props) {
+  const user = useSelector(({ User }) => User.data.user);
+  const [urlLink, setUrlLink] = useState("#");
+
+  useEffect(() => {
+    if (user.email.includes("navgurukul")) {
+      axios({
+        method: METHODS.GET,
+        urlLink: `${process.env.REACT_APP_SARAL_URL}/${user.email}`,
```

Comment:
urlLink will be undefined. Please check other API calls. It's `url` and not `urlLink`

Reply:
@komalng Please use redux for API calls.

Reply:
https://github.com/navgurukul/bhanwari-devi/pull/79/commits/969ef1f349f4653739a04c4f5d285267183c111f